### DISCUSSION TITLE
Looking for deck.gl developer for modifying and further development of an existing webapp

### DISCUSSION THREAD

**Discussion by csandor** [visgl/deck.gl#5555] [Q&A]
I'm not sure if this is appropriate here, but if anyone from here would be up for a smaller T&M based project (with bright future of course ;) ) in modifying and further developing a deck.gl map view in a webapp please contact me at sandor.csaba@gmail.com.

The task would include:
- creating visual controls for moving the camera with high precision (instead of mouse interaction) pan-tilt-zoom
- reset view control
- creating several static camera positions and a widget to switch between them

**Comment by ibgreen** [comment]
https://docs.unfolded.ai/development/services
